include/inviwo/core/datastructures/camera.h

```diff
@@ -255,43 +269,41 @@ class IVW_CORE_API OrthographicCamera final : public Camera {
  */
 class IVW_CORE_API SkewedPerspectiveCamera final : public Camera {
 public:
-    SkewedPerspectiveCamera(vec3 lookFrom = vec3(0.0f, 0.0f, 2.0f), vec3 lookTo = vec3(0.0f),
-                            vec3 lookUp = vec3(0.0f, 1.0f, 0.0f), float nearPlane = 0.01f,
-                            float farPlane = 10000.0f, float fieldOfView = 60.f,
-                            float aspectRatio = 1.f, vec2 frustumOffset = vec2(0.0f, 0.0f));
+    SkewedPerspectiveCamera(
+        vec3 lookFrom = cameradefaults::lookFrom, vec3 lookTo = cameradefaults::lookTo,
+        vec3 lookUp = cameradefaults::lookUp, float nearPlane = cameradefaults::nearPlane,
+        float farPlane = cameradefaults::farPlane, float fieldOfView = cameradefaults::fieldOfView,
```

Comment:
Think 60 was just a bad choice to begin with. 38 is recommended by Martin :)

Reply:
It was 60 also for the perspective one, so it should not have anything to do with the stereo aspects

Reply:
http://gameaccessibilityguidelines.com/if-the-game-uses-field-of-view-3d-engine-only-set-an-appropriate-default-for-the-expected-viewing-environment/

60 is default in Unity as well

Reply:
Just some other input on the question
https://forum.unity.com/threads/what-is-a-normal-field-of-view.70202/
” For PC games, the screen aspect is important, because players sit roughly at the same distance from the monitor. Let's say now you're using a 21:9 monitor, compared to a 16:9 one, you should see more space horizontally, NOT vertically, right? But in Unity, the aspect remains the same! So if you increase the FOV, you see more space horizontally AND vertically, that's the reason why developers feel weird and uncomfortable when they set the value to 90 or even higher on their camera. In fact, a higher FOV in Unity will make you feel like you're traveling at the speed of light, not only "wider view".

Last question, how to see wider? Change the aspect instead.”

Reply:
I think, there is a big difference between a 3D game (like a FPS) and a scientific visualization. I find that 60° create ways too much distortion. @martinfalk might also have opinions here. Also you can always change the camera to fit your preference...

Reply:
sketchup uses a default of 35°

Reply:
A vertical fov of 38° is closer to human vision. Notice the extreme perspective distortions happening at 60 degrees. And I am with Peter here that visualization is _not_ the same as games. In games you want to see as much of your surroundings as possible. Here we mostly have a tiny (square) canvas showing one dataset. Another thing to consider: to be more accurate one should actually use orthographic. But that messes with 3D perception of shapes/objects.

Reply:
and orthographic "zoom" (changing width) now also works with the mouse